sled-agent/src/services.rs

```diff
@@ -3161,7 +3163,7 @@ impl ServiceManager {
         let services = match self.inner.sled_mode {
             // A pure gimlet sled should not be trying to activate a switch
```

Comment:
nit: maybe the comment should also be updated?

Reply:
Done.  I also removed the passive voice.